spec/components/schemas.yml

```diff
@@ -113,7 +113,6 @@ components:
       maxItems: 4
       minItems: 4
       type: array
-      readOnly: true
```

Comment:
hmm, shouldn't this stay readonly, since e.g. in https://service-stac.dev.bgdi.ch/api/stac/static/spec/v0.9/apitransactional.html#operation/getFeature it's readonly.
shouldn't `/search` path rather reference `bboxFilter` schema?

Reply:
Yes, the point in that ticket is, that here: 
https://service-stac.dev.bgdi.ch/api/stac/static/spec/v0.9/apitransactional.html#operation/postSearchSTAC
there is no `bbox` listed in the `REQUEST BODY SCHEMA` table on the left, but on the right, in the `Request samples` there's `bbox` in the sample payload.

Reply:
Sorry for my earlier comment here, it was too early obviously. These current changes seem to work.
Currently there's already a reference to `"#/components/schemas/searchBody"` in the `requestBody` of the `/search` `post` part. And the `searchBody` already references the `bboxFilter`. Not sure, why it does not appear, although it should..